feature/02_create_account.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Create an account
+
+    As a common user
+    I want to Create an account
+    In order to have an user account
+
+    Scenario: Create an account
+        Given the user is on demoblaze page
+        When the user complete the Sign Up form
+        Then confirm message is displayed
```

Comment:
minor: add an empty line to the end of the file.
Notice how github shows a red symbol when you don't.
if you wanna know more about this convention, take a look at this: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline#:~:text=The%20way%20Unix%20regards%20its,an%20%5Cn%20to%20end%20it.